examples/PZEMHardSerial/PZEMHardSerial.cpp

```diff
@@ -33,18 +23,26 @@ pins.
  * The ESP32 HW Serial interface can be routed to any GPIO pin 
  * Here we initialize the PZEM on Serial2 with RX/TX pins 16 and 17
  */
+#if !defined(PZEM_RX_PIN) && !defined(PZEM_TX_PIN)
+#define PZEM_RX_PIN 16
+#define PZEM_TX_PIN 17
+#endif
+
+#define PZEM_SERIAL Serial2
```

Comment:
Here, and bellow, it's always set explicitly. No option to set it by compiler flags